xdsl/dialects/tensor.py

```diff
@@ -41,37 +43,6 @@
 from xdsl.printer import Printer
 from xdsl.traits import NoMemoryEffect
 from xdsl.utils.exceptions import VerifyException
-from xdsl.utils.hints import isa
-
-
-@dataclass(frozen=True)
-class ContiguousArrayOfIntArray(AttrConstraint):
```

Comment:
What's the motivation for moving this? Is it in a utils file in MLIR? Do you envision this being useful for other dialects?

Reply:
OK let's do that in a separate PR then? WIll make it easier to review. I think in that case it might be good to have more documentation both for the file and each function.

Reply:
sure, will do!